### PR TITLE
CRB-142: Add CI job to build container and verify healtcheck

### PR DESCRIPTION
**Pull request by atodorov**

### CODE REVIEW

File: .github/workflows/container.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    - name: Build container image
+      run: |
+        docker build -t gluwa/container-under-test PoW/
+
+    - name: Execute container image
+      run: |
+        docker run -d --rm --name container-under-test gluwa/container-under-test
+
+    - name: Verify healtcheck
```
Comment (seandeneen-gluwa) [inline .github/workflows/container.yml]: Minor typo here.

Comment (ghost) [reply .github/workflows/container.yml]:
```suggestion
    - name: Verify healthcheck
```

File: .github/workflows/container.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      run: |
+        docker build -t gluwa/container-under-test PoW/
+
+    - name: Execute container image
+      run: |
+        docker run -d --rm --name container-under-test gluwa/container-under-test
+
+    - name: Verify healthcheck
+      run: |
+        docker inspect -f '{{.State.Health.Status}}' container-under-test
```
Comment (seandeneen-gluwa) [inline .github/workflows/container.yml:26]: One of the checks are failing with the following message:

`Template parsing error: template: :1:8: executing "" at <.State.Health.Status>: map has no entry for key "Health"`

Is this expected?

**Review by seandeneen-gluwa — CHANGES_REQUESTED** [review]

File: .github/workflows/container.yml (hunk 2 above)
Comment (atodorov) [reply .github/workflows/container.yml:26]: yes, b/c the container doesn't provide a command for a health check. IDK what that command should be but I'm happy to implement it if somebody can decide on it.

**Pull request closed without merging**